addons/website_event/controllers/main.py

```diff
@@ -298,11 +298,11 @@ def _process_attendees_form(self, event, form_details):
             {'1-name': 'r', '1-email': 'r@r.com', '1-phone': '', '1-event_ticket_id': '1'}
         """
         registration_fields = request.env['event.registration']._fields
-
         registrations = {}
         global_values = {}
         for key, value in form_details.items():
-            counter, field_name = key.split('-', 1)
+            counter, attr_name = key.split('-', 1)
+            field_name = attr_name.split('-')[0]
```

Comment:
I would prefer sticking more like the old implementation until we have something nicer, like
 * don't change it (we don't receive 1-answer_ids-3 in website_event)
 * re-check form_details to extract answer_ids in website_event_question instead of working with super result